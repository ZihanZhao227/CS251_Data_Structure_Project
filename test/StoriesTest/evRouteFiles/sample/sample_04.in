10
0 0 0 CITY StartCity
1 35 0 CHARGING_STATION Charging1
2 70 0 CHARGING_STATION Charging2
3 50 20 RESTAURANT Restaurant1
4 105 0 CHARGING_STATION Charging3
5 90 15 TOURIST_ATTRACTION Museum1
6 140 0 CHARGING_STATION Charging4
7 120 -20 RESTAURANT Restaurant2
8 175 0 CHARGING_STATION Charging5
9 210 0 CITY DestinationCity