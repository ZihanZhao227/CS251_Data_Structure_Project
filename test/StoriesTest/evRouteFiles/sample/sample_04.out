0 9 40.0 210.000
0 1 2 4 6 8 9